{{cookiecutter.project_repo}}/.github/workflows/pythonpackage.yml

```diff
@@ -9,21 +9,40 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8]
       fail-fast: false
 
     steps:
       - uses: actions/checkout@v2
+
       - name: Set up Python ${% raw %}{{ matrix.python-version }}{% endraw %}
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2
         with:
           python-version: ${% raw %}{{ matrix.python-version }}{% endraw %}
-      - name: Install dependencies
+
+      - name: Install dependencies and disable IPv6
         run: |
-          python -m pip install --upgrade pip
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python - --version 1.0.0
-          echo "$HOME/.poetry/bin" >> $GITHUB_PATH
+          sudo snap install dotnet-sdk --classic
+          sudo snap alias dotnet-sdk.dotnet dotnet
+          sudo add-apt-repository -y ppa:ubuntugis/ppa
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev
+          sudo apt-get install -y --no-install-recommends libffi-dev
+          sudo apt-get install -y --no-install-recommends lzma liblzma-dev
+          sudo apt-get install -y --no-install-recommends libgeos-dev libgeos++-dev
+          sudo apt-get install -y --no-install-recommends libproj-dev proj-data proj-bin
+          sudo apt-get install -y --no-install-recommends libsnappy-dev
+          sudo apt-get install -y --no-install-recommends python3-h5py
+          sudo apt-get install -y --no-install-recommends gdal-bin libgdal-dev python3-gdal
+          python -m pip install --upgrade pip poetry
+
       - name: Run tox
+        env:
+          DOTNET_ROOT: /snap/dotnet-sdk/current
         run: |
-          poetry install
+          poetry config repositories.ukhoget https://pkgs.dev.azure.com/ukhogov/_packaging/ukhoget/pypi/simple/
+          poetry config http-basic.ukhoget SVC_github-ci ${% raw %}{{ secrets.UKHO_BOT_UKHOGET_FEED_PAT }}{% endraw %}
+          poetry export -f requirements.txt --dev --with-credentials --without-hashes -o requirements.txt
```

Comment:
If there are any Extras (i.e. -E options on the poetry install) they also need to be included here. Difficult to automate generation of this though. I'll be testing this on gsfpy-cleanse so will keep you updated.

Reply:
fail on first run with info message that tells the user that they'll need to add their extras